hw/dv/tools/dvsim/common_sim_cfg.hjson

```diff
@@ -67,7 +67,12 @@
                "+define+UVM_REG_DATA_WIDTH={tl_dw}",
                "+define+UVM_REG_BYTENABLE_WIDTH={tl_dbw}",
                "+define+SIMULATION",
-               "+define+DUT_HIER={dut_instance}"]
+               "+define+DUT_HIER={dut_instance}",
+               // Introduce compile-time randomization. Use `BUILD_SEED in Verilog sources to set
+               // constants that have no side-efects, such as addition or deletion of logic. Useful
+               // to mutate fixed device values such as secret keys, identification constants, etc.
+               // across successive simulation runs.
```

Comment:
It might be useful to note here that the value is by default a 32bit value (unless overridden explicitly to a larger value).

Reply:
nit: side-efects -> side-effects

Reply:
Done, and done.